be/src/exec/vectorized/es_http_components.cpp

```diff
@@ -525,6 +525,10 @@ Status ScrollParser::_append_date_val(const rapidjson::Value& col, Column* colum
             if (!value.from_string(raw_str, val_size)) {
                 RETURN_ERROR_IF_CAST_FORMAT_ERROR(col, type);
             }
+            // https://en.wikipedia.org/wiki/ISO_8601
```

Comment:
if here give a simple example, the code will be more clear

Reply:
Is date required to convert timezone?

Reply:
date was unnecessary. date was treat as 'YYYY-MM-DD'

Reply:
fixed